syncd/syncd.cpp

```diff
@@ -2907,6 +2907,17 @@ void processFlexCounterEvent(
 
                 FlexCounter::setPriorityGroupAttrList(vid, rid, groupName, pgAttrIds);
             }
+            else if (objectType == SAI_OBJECT_TYPE_SWITCH && field == SWITCH_SENSOR_ID_LIST)
```

Comment:
why we need this here as special case ?
also this solution for those cases here is getting bigger and bigger we should think of some generic approach